warn-stealers.lua

```diff
@@ -0,0 +1,107 @@
+-- Script to warn when creatures that may steal food become visible
```

Comment:
also consider supporting `enable` syntax. See https://github.com/DFHack/scripts/pull/405/files for an example.